app/models/service_template_transformation_plan_task.rb

```diff
@@ -280,6 +280,10 @@ def kill_virtv2v(signal = 'TERM')
 
     _log.info("Killing conversion pod for task '#{id}'.")
     conversion_host.kill_virtv2v(id, signal)
+  rescue => err
+    _log.error("Couldn't kill conversion pod for task '#{id}': #{err.message}")
+    update_options(:virtv2v_finished_on => Time.now.utc.strftime('%Y-%m-%d %H:%M:%S'))
```

Comment:
Super minor, but `.strftime('%Y-%m-%d %H:%M:%S')` is equivalent to `.strftime('%F %T')`